models/kyberswap/arbitrum/kyberswap_arbitrum_trades.sql

```diff
@@ -0,0 +1,234 @@
+        ,''                                                                AS maker
+        ,returnAmount                                                      AS token_bought_amount_raw
+        ,spentAmount                                                       AS token_sold_amount_raw
+        ,cast(NULL as double)                                              AS amount_usd
+        ,dstToken                                                          AS token_bought_address
+        ,srcToken                                                          AS token_sold_address
+        ,contract_address                                                  AS project_contract_address
+        ,evt_tx_hash                                                       AS tx_hash
+        ,''                                                                AS trace_address
+        ,evt_index
+    FROM {{ source('kyber_arbitrum', 'AggregationRouterV3_evt_Swapped') }}
+    WHERE
+        {% if is_incremental() %}
+        evt_block_time >= date_trunc("day", now() - interval '1 week')
+        {% else %}
+        evt_block_time >= '{{ project_start_date }}'
+        {% endif %}
+
+    UNION ALL
+
+    SELECT
+        'v4'                                                               AS version
+        ,evt_block_time                                                    AS block_time
+        ,sender                                                            AS taker
+        ,''                                                                AS maker
+        ,returnAmount                                                      AS token_bought_amount_raw
+        ,spentAmount                                                       AS token_sold_amount_raw
+        ,cast(NULL as double)                                              AS amount_usd
+        ,dstToken                                                          AS token_bought_address
+        ,srcToken                                                          AS token_sold_address
+        ,contract_address                                                  AS project_contract_address
+        ,evt_tx_hash                                                       AS tx_hash
+        ,''                                                                AS trace_address
+        ,evt_index
+    FROM {{ source('kyber_arbitrum', 'MetaAggregationRouter_evt_Swapped') }}
+    WHERE
+        {% if is_incremental() %}
+        evt_block_time >= date_trunc("day", now() - interval '1 week')
+        {% else %}
+        evt_block_time >= '{{ project_start_date }}'
+        {% endif %}
+)
+
+SELECT
+    'arbitrum'                                                           AS blockchain
+    ,'kyberswap'                                                         AS project
+    ,kyberswap_dex.version                                               AS version
+    ,SELECT (CASE
```

Comment:
Hi, the last avalanche reviews actually apply to all models on all chains. You may miss to remove a `SELECT` here.
Moreover you can use [dbt-test result](https://github.com/duneanalytics/spellbook/actions/runs/3592028878/jobs/6047272218#step:11:32) as a hint to debug/fix the code. As we may need to pass the dbt-test before the PR is ready for the final review.